src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -137,6 +134,6 @@ public RobotMap() {
 		HumanInput.Operator.stick.setDeadzone(Constant.HumanInput.OPERATOR_JOYSTICK_MINIMUM_THRESHOLD);
 		HumanInput.Driver.xbox = new CustomXbox(Port.HumanInput.xboxController);
 		HumanInput.Driver.xbox.setDeadZone(RobotMap.Constant.HumanInput.XBOX_MINIMUM_THRESHOLD);
-		Component.mainSubsystems = new Subsystem[] {Component.ballIntake, Component.ballDumper, Component.flywheel};
+		Component.mainSubsystems = new Subsystem[] {Component.flywheel, Component.ballIO};
```

Comment:
Nice job! This will be helpful in competition – all of the main subsystems' current commands are shown on SmartDashboard.